srcpkgs/irssi-otr/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'irssi-otr'
+pkgname=irssi-otr
+version=1.0.2
+revision=1
+build_style=gnu-configure
+configure_args="--with-gnu-ld
+ --with-irssi-headers=${XBPS_CROSS_BASE}/usr/include/irssi
+ --oldincludedir=${XBPS_CROSS_BASE}/usr/include"
+hostmakedepends="autoconf automake libtool m4 pkg-config"
+makedepends="glib-devel irssi-devel libgcrypt-devel libotr-devel"
```

Comment:
can be libglib-devel

Reply:
https://travis-ci.org/void-linux/void-packages/jobs/456436582

Reply:
i see, carry on then